### ISSUE TITLE
a1ad/mikrotik-logs : failed auth not parsed on ROS 7.9

### ISSUE THREAD

**Issue by VA2XJM** [crowdsecurity/hub#726]
@a1ad 

Seems that the new format to display failed auth is now as follow: `denied winbox/dude connect from XXX.XXX.XXX.XXX`

**Comment by VA2XJM** [comment]
Ok, after more tests, both the orignal synthax and the one in first post are used at the same time... The "denied winbox/dude" is when a connection attempt is made from an incompatible client. That could be an interesting addition.

**Comment by a1ad** [comment]
@VA2XJM Is that a full log line?

**Comment by VA2XJM** [comment]
@a1ad Affirmative!
![image](https://github.com/crowdsecurity/hub/assets/13443249/257d9fee-bfae-4acf-b8fb-1d7fa4cb8a4c)

**Comment by LaurenceJJones** [comment]
So that is the full log line? cause from image we can see a date and warning?

**Comment by a1ad** [comment]
No, that is not a syslog line. You need to connect the Tik to a remote syslog server.

**Comment by VA2XJM** [comment]
I will check next time I am on location (no remote access), I'm relaying the info only, but the owner says this is the complete line in the log file (rsyslogd dump to a fil so Crowdsec can read it).

I'll get back to you sometime next week.
